.github/COMMUNITY_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Community engineering issue
+about: Create an issue related to a community engineering effort
+---
+
+# Community engineering issue
+
+<!-- (REQUIRED) Provide information for the doc request including any Community code issues or PRs, Magento versions, or user guide pages. -->
+
+Type of Doc:
+
+-  [ ] New topic
+-  [ ] Topic Update, user guide link:
+-  [ ] Other
+
+Magento 2 GitHub Issue: https://github.com/magento/merchdocs/issues/link
+
+Magento 2 GitHub PR: https://github.com/magento/merchdocs/pull/link
+
+Magento 2 version:
+
+-  [ ] 2.3.X
+-  [ ] 2.4.X
+-  [ ] back port
```

Comment:
it is unclear in what cases do I use `back port` or `forward port`, since I can specify the versions I need